dist/backdraft.js

```diff
@@ -1476,6 +1544,70 @@ _.extend(Plugin.factory, {
       this.trigger("draw", arguments);
     },
 
+    exportData : function(sUrl) {
+      var oSettings = this.dataTable.fnSettings;
+      var aoData = this.dataTable._fnAjaxParameters( oSettings );
+      this._addServerParams( aoData );
+      this._fetchCSV( sUrl, aoData );
+    },
+
+    _fetchCSV : function (sUrl, aoData) {
+      if (this.serverSideFiltering) {
+        var filterJson = {};
+        filterJson.name = "ext_filter_json";
+        filterJson.value = this._getFilteringSettings();
+        aoData.push(filterJson);
+      }
+      $.ajax({
+        url: sUrl,
+        data : aoData,
+        dataType : "text",
+        cache : false,
+        type : this.ajaxMethod || "GET",
+        success: function(response, status, xhr) {
```

Comment:
exportData feels more like an application concern at this point (ie maybe should live in actionTable in our repo?)  But moreso I would like to see most of this code removed in favor of just an interface that gets a full URL with all the params, and we then send that off to the server with a .csv extension..